bin-src/git/git.ts

```diff
@@ -211,6 +211,15 @@ export async function checkout(ref: string) {
   return execGitCommand(`git checkout ${ref}`);
 }
 
+export async function checkoutFile(
+  ref: string,
+  fileName: string,
+  targetFileName = `${ref}.${fileName}`
```

Comment:
Shouldn't this be in a temp dir or something?

Reply:
Ah yes, of course 👍 Wouldn't want to clobber up their working dir.